flocker/node/agents/test/test_blockdevice.py

```diff
@@ -1337,6 +1350,29 @@ def test_resize_destroyed_volume(self):
         )
         self.assertEqual(exception.args, (volume.blockdevice_id,))
 
+    def assertVolumesDistinct(self, block_device_api2):
```

Comment:
And perhaps this would be easier to read if it was a free function...something like

`assert_volumes_distinct(test_case, api1, api2)`